core/src/utils/test/playwright/drag-element.ts

```diff
@@ -21,14 +30,17 @@ export const dragElementBy = async (
   const startX = startXCoord === undefined ? boundingBox.x + boundingBox.width / 2 : startXCoord;
   const startY = startYCoord === undefined ? boundingBox.y + boundingBox.height / 2 : startYCoord;
 
-  const endX = startX + dragByX;
-  const endY = startY + dragByY;
-
+  // Navigate to the start position.
   await page.mouse.move(startX, startY);
+
   await page.mouse.down();
 
-  await page.mouse.move(endX, endY, { steps: 10 });
-  await page.mouse.up();
+  // Drag the element.
+  await moveElement(page, startX, startY, dragByX, dragByY);
+
+  if (releaseDrag) {
+    await page.mouse.up();
+  }
```

Comment:
Why do we conditionally release the drag gesture? I'd expect the gesture to always be released when it's done.

Reply:
This is prep work for when range tests are enabled. One of the range tests is to check that [scroll cannot occur while the knob is swiped](https://github.com/ionic-team/ionic-framework/pull/27808/files#diff-4cc6a4738b72ca53b59df31fc34e2e29e5f510dde3ff52dad323c1b7bedf59d1R80). The test needs to do the check while the knob is being held down and moving.